python/GafferUI/CompoundEditor.py

```diff
@@ -491,3 +491,9 @@ def __drop( self, tabbedContainer, event ) :
 	def __removeCurrentTab( self ) :
 
 		self.remove( self.getCurrent() )
+
+	def __removePanel( self ) :
+
+		splitContainer = self.ancestor( _SplitContainer )
+		splitContainerParent = splitContainer.parent()
+		GafferUI.EventLoop.addIdleCallback( functools.partial( Gaffer.WeakMethod( splitContainerParent.join ), 1 - splitContainerParent.index( splitContainer ) ) )
```

Comment:
Could you add a comment here explaining why we're deferring the call?

Reply:
Squashed in a comment (and todo) in 624ed1b

Reply:
Sorry, in 54c7e3c